addons/exact-count/studio.js

```diff
@@ -3,48 +3,31 @@ export default async function({
   global,
   console
 }) {
-  function countProjects(url, page, delta, callback) {
+  function countProjects(url, offset, delta, callback) {
     const request = new XMLHttpRequest();
-    request.open("GET", url + page + "/");
+    request.open("GET", url + offset);
     request.onreadystatechange = function() {
       if (request.readyState == 4) {
-        if (request.status == 200) {
-          countProjects(url, page + delta, delta, callback);
-        } else if (request.status == 404) {
-          if (page != 1) {
-            if (delta == 1) {
-              const correctReq = new XMLHttpRequest();
-              correctReq.open("GET", url + (page - 1) + "/");
-              correctReq.onreadystatechange = function() {
-                if (correctReq.readyState == 4) {
-                  if (correctReq.status == 200) {
-                    const parser = new DOMParser();
-                    const list = parser.parseFromString(
-                      "<ul id='projects'>" + correctReq.responseText + "</ul>", "text/html"
-                    );
-                    let count = 60 * (page - 2);
-                    count += list.querySelector("#projects").querySelectorAll("ul > li").length;
-                    callback(count);
-                  }
-                }
-              };
-              correctReq.send();
-            } else {
-              page -= delta;
-              delta /= 10;
-              countProjects(url, page + delta, delta, callback);
-            }
-          }
+        let pageLen = JSON.parse(request.response).length
+        if (pageLen == 20) {

```

Comment:
You can actually set limit to 40 and go 40 by 40, instead of 20 by 20

Reply:
Didn't even realize lol